cmd/kube-apiserver/app/server.go

```diff
@@ -139,9 +139,10 @@ cluster's shared state through which all other components interact.`,
 // Run runs the specified APIServer.  This should never exit.
 func Run(ctx context.Context, opts options.CompletedOptions) error {
 	// To help debugging, immediately log version
-	klog.Infof("Version: %+v", version.Get())
+	logger := klog.FromContext(ctx)
```

Comment:
I'm curious about one thing. Do we only modify the cmd/kube-apiserver package?

Reply:
@googs1025 
Thank you for asking.
We plan to support not only cmd/kube-apiserver but also all packages.

cf. 
For cmd/genswaggertypedocs https://github.com/kubernetes/kubernetes/pull/127722
For cmd/fieldnamedocscheck https://github.com/kubernetes/kubernetes/pull/127708